themes/finna2/templates/ajax/feed-list.phtml

```diff
@@ -1,6 +1,6 @@
 <!-- START of: finna - ajax/feed-list.phtml -->
 <div class="list-feed<?= $this->images ? ' with-image' : ''; ?>">
-<?php if ($this->title || $this->translateTitle): ?><h2 class="feed-header"><?= $this->translateTitle ? $this->transEsc($this->translateTitle) : $this->escapeHtml($this->title) ?></h2><?php endif; ?>
+<?php if (!$this->translateTitle): ?><h2 class="feed-header"><?= $this->escapeHtml($this->title) ?></h2><?php endif; ?>
```

Comment:
Tähän asti oli kaikki selvää, mutta nyt jäin ihmettelemään, miksei titleä enää näytetä, jos translateTitle=true.

Reply:
Koska se configin translateTitle olisi elementtinä aina siellä ulkopuolella eli jo feed.phtml. Niin sitten sitä titleä, joka muodostetaan myöhemmin ei tarvitse näyttää, koska se title on jo haettu ja näkyvissä, koska tullut sieltä config.

Reply:
Lisäisitkö sitten vielä selventävän kommentin tähän? Jotain tyyliin:
`// Translated title is displayed in the container, so ignore it here`